thoth/adviser/wraps/pypi_release.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # thoth-adviser
-# Copyright(C) 2020 - 2021 Fridolin Pokorny
+# Copyright(C) 2021 Fridolin Pokorny
```

Comment:
@KPostOffice if you are adding significant amout of code, please feel free to add yourself here ;)

Reply:
Thanks for your suggestion, Christoph.